spec/views/organisation_reports/without_users_index.html.erb_spec.rb

```diff
@@ -1,15 +0,0 @@
-require 'rails_helper'
```

Comment:
deleting view specs is great but we should always double check that we have the relevant acceptance tests in place to provide cover